scripts/build/electron-installer-debian-linux.sh

```diff
@@ -25,9 +25,7 @@ if [[ "$OS" != "Linux" ]]; then
   exit 1
 fi
 
-ELECTRON_INSTALLER_DEBIAN="./node_modules/.bin/electron-installer-debian"
-
-./scripts/build/check-dependency.sh "$ELECTRON_INSTALLER_DEBIAN"
+./scripts/build/check-dependency.sh electron-installer-debian
```

Comment:
Haha, funny how things go around in circles ;-) ( https://github.com/resin-io/etcher/commit/880ca257a673da94d6c58c67c43d124a0be74a35#diff-d3e903f23548d60d01b1c17f94913c6e )